test/.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Remove the text attribute from reference files, so that git doesn't convert
+# line separators on Windows machines. It causes the index files to become out
+# of sync with the fasta files.
+*.fa* -text
```

Comment:
Put this in the top-level _.gitattributes_ file.  But wouldn't it be better to regenerate the _.fai_ files instead?